### DISCUSSION TITLE
Embedded Dashboards don't load consistently

### DISCUSSION THREAD

**Discussion by blokkies48** [apache/superset#28049] [Q&A / Help] [answered]
I am currently tasked with integrating Superset dashboards into our company's system. However, I have encountered an issue where the dashboard most of the time fails to load completely, often displaying the Superset loading icon indefinitely.

The environment in which I am working comprises Superset version 2.1.0, Django version 4.2, and Python version 3.11. Additionally, I am utilizing the JavaScript SDK import within a Django project.

I would greatly appreciate any guidance or advice on resolving this matter.

![image](https://github.com/apache/superset/assets/66180897/57692d9d-6f5b-43d1-b09f-63a7e841cc7b)

**Comment by blokkies48** [comment, marked as answer]
I believe my error is with the load balancer on AWS and not Superset.

**Comment by Prachil-MI** [comment]
Hello @blokkies48, 

Can you please elaborate on the issue that you faced with the AWS LB and **what solution did you apply**? How was this LB connected to the Superset Server/Dashboard integration?
